pkgs/applications/office/ledger-autosync/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, python3Packages, fetchFromGitHub, ledger, hledger }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ledger-autosync";
+  version = "1.0.0";
+# no tests included in PyPI tarball
+  src = fetchFromGitHub {
+    owner = "egh";
+    repo = "ledger-autosync";
+    rev = "v${version}";
+    sha256 = "1fn32c02idccdmf9906pxn248qc9basjy2kr2g600806k3qvw84a";
+  };
+
+  propagatedBuildInputs = with python3Packages; [  
+    asn1crypto  
+    beautifulsoup4  
+    cffi  
+    cryptography  
+    entrypoints  
+    fuzzywuzzy  
+    idna  
+    jeepney  
+    keyring  
+    lxml  
+    mock  
+    nose  
+    ofxclient  
+    ofxhome  
+    ofxparse  
+    pbr  
+    pycparser  
+    secretstorage  
+    six  
+  ];  
+  
+  # Checks require ledger as a python package,
+  # ledger does not support python3 while ledger-autosync requires it.
+  checkInputs = with python3Packages; [ ledger hledger nose mock ];
+  checkPhase = ''
```

Comment:
Can we get a comment here also stating that we've omitted the `ledger-python` tests because they're broken?
It's for the same reason stated above but it's not obvious we're omitting a test here.